Packs/CofenseTriage/Integrations/CofenseTriagev2/CofenseTriagev2.py

```diff
@@ -393,6 +518,63 @@ def search_reports(
     return matches
 
 
+def search_inbox_reports_command(triage_instance) -> None:
+    reported_at = parse_date_range(demisto.args().get("reported_at", "7 days"))[
+        0
+    ].replace(tzinfo=timezone.utc)
+
+    reporters_clause = build_reporters_clause(triage_instance)
+    if reporters_clause is None:
+        return_outputs("Reporter not found.", {}, {})
+        return
+
+    reports = TriageInboxReports(
+        triage_instance,
+        start_date=reported_at,
+        filter_params={
+            "subject": demisto.getArg("subject"),
+            "url": demisto.getArg("url"),
+            "file_hash": demisto.getArg("file_hash"),
+            **reporters_clause,
+        },
+        max_pages=int(demisto.getArg("max_matches") or 10)
+        / 10,  # TODO Triage's number of items per page
+    ).inbox_reports()
+
+    if not reports:
+        return_outputs("No results found.", {}, {})
+        return
+
+    reports_dict = [report.to_dict() for report in reports]
+    ec = {
+        "Cofense.Report(val.ID && val.ID == obj.ID)": snake_to_camel_keys(reports_dict)
+    }
+    hr = tableToMarkdown(
+        "Reports:", reports_dict, headerTransform=split_snake, removeNull=True
+    )
+
+    return_outputs(hr, ec, reports_dict)
+
+
+def build_reporters_clause(triage_instance):
+    reporter_address_or_id = demisto.getArg("reporter")
+
+    if not reporter_address_or_id:
+        return {}
+
+    if reporter_address_or_id.isdigit():
+        return {"reporter_ids": [int(reporter_address_or_id)]}
+
+    reporters = TriageReporters(
+        triage_instance, email=reporter_address_or_id
+    ).reporters()
+
+    if len(reporters) == 0 or not any([reporter.exists() for reporter in reporters]):
+        return None
```

Comment:
why not empty dict like in before?
```suggestion
        return {}
```

Reply:
`{}` signifies that the user has not requested any reporter filter. This is not an error.

`None` signifies that the user has requested a filter, but there are no reporters matching it. This is an error.

I've pushed another commit refactoring this to raise and catch an exception instead of returning `None`.